.github/workflows/not-earthly.yml

```diff
@@ -0,0 +1,226 @@
+name: Not earthly ci
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+      - master
+  workflow_dispatch:
+    inputs:
+      upload:
+        description: "Upload the container to our registry"
+        default: false
+        type: boolean
+
+env:
+  AWS_REGION: "eu-central-1"
+  SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    if: github.event.action != 'closed' || github.event.pull_request.merged == true
+    concurrency:
+      group: pr-${{ github.event.pull_request.number }}-author-${{ github.event.pull_request.user.login }}
+      cancel-in-progress: true
+    permissions:
+      id-token: write
+      contents: write
+    steps:
+      - name: Checkout partner-chains
+        uses: actions/checkout@v4
+
+      - name: Checkout sidechains-infra-priv
+        uses: actions/checkout@v4
+        with:
+          repository: input-output-hk/sidechains-infra-priv
+          token: ${{ secrets.ACTIONS_PAT }}
+          path: sidechains-infra-priv
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y protobuf-compiler
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+          source $HOME/.cargo/env
+          rustup target add x86_64-unknown-linux-gnu
+
+      - name: Build partner-chains-node
+        run: |
+          cargo build -p partner-chains-node --locked --release --target x86_64-unknown-linux-gnu
+          cp target/x86_64-unknown-linux-gnu/release/partner-chains-node partner-chains-node
+
+      - name: Transform YAML to JSON and Build chain specs
+        run: |
+          mkdir -p configs
+          yq eval-all -i '. as $item ireduce ({}; . * $item )' sidechains-infra-priv/src/kube/substrate-poc/environments/helm/substrate-node-stack-chart/values/chains/*.yaml -o=json
```

Comment:
Does Earthly use it as a source of truth?

Reply:
Earthfile uses values from this repo.
```
chainspecs:
  FROM +setup
  DO +INSTALL
  COPY devnet/.envrc devnet/.envrc
  COPY devnet/addresses.json devnet/addresses.json
  COPY staging/.envrc staging/.envrc
  COPY staging/addresses.json staging/addresses.json
```
Why this change?

Reply:
It does not use this?

https://github.com/input-output-hk/sidechains-infra-priv/blob/master/Earthfile

Reply:
Yes you are correct

Reply:
This is Earthlfile in infra-priv repo. We have problems with Earthfile in our repo. I don't see reference to sidechains-infra-priv in Earthfile nor in earthly.yml.